### DISCUSSION TITLE
3D model of person not going well

### DISCUSSION THREAD

**Discussion by Groeneg** [alicevision/Meshroom#2001] [Q&A]
I have taken about 50 pictures of a person to have a clear view of him. Putting this into Meshroom, didn't give me a 3D model of it. Only the photos from the front were used, but I took photos around that person. I have done it with a small statue and that worked fine. the goal is to produce a tiny copy of themselves

Could it be that I didn't had the correct settings at the start?
![IMG_20230423_102824344](https://user-images.githubusercontent.com/51157662/235314851-9740acb0-90fa-4058-a2ce-67f05bc37bd2.jpg)

I use Meshroom 2023.1.0

Looking forward to some tips.

**Comment by natowi** [comment]
Sorry for the late reply. Capturing humans is not an easy task. Slight movements can mess up the reconstruction. Most comes down to the capturing process.
https://www.youtube.com/watch?app=desktop&v=bX_1q0N-EHE
